frontend/src/pages/NewQuestion/index.tsx

```diff
@@ -71,57 +44,60 @@ const NewQuestion = () => {
         toast.error(`${file.name} is more than 5MB`);
         continue;
       }
+      formData.append("images[]", file);
+    }
 
-      const blob = new Blob([file], { type: file.type });
-
-      // this parts onwards should be in the backend...
-      const storageRef = ref(storage, uuidv4());
-      const uploadTask = uploadBytesResumable(storageRef, blob);
-
-      uploadTask.on(
-        "state_changed",
-        (snapshot) => {
-          // Observe state change events such as progress, pause, and resume
-          // Get task progress, including the number of bytes uploaded and the total number of bytes to be uploaded
-          const progress = (snapshot.bytesTransferred / snapshot.totalBytes) * 100;
-          console.log("Upload is " + progress + "% done");
-          switch (snapshot.state) {
-            case "paused":
-              console.log("Upload is paused");
-              break;
-            case "running":
-              console.log("Upload is running");
-              break;
-          }
-        },
-        (error) => {
-          console.error(error);
-          toast.error("Upload of images failed");
-        },
-        () => {
-          getDownloadURL(uploadTask.snapshot.ref).then((downloadURL) => {
-            setUploadedImagesUrl((prevUrls) => [...prevUrls, downloadURL]);
-          });
-        }
-      );
+    try {
+      const response = await fetch("http://localhost:3000/api/questions/images", {
```

Comment:
i think there's an axios instance under the `utils` folder to send requests to the backend

Reply:
fixed!